rholang/src/main/scala/coop/rchain/rholang/interpreter/dispatch.scala

```diff
@@ -68,13 +61,13 @@ object RholangAndScalaDispatcher {
     (dispatcher, reducer)
   }
 
-  def create2[M[_]: Sync: Parallel: _cost](
+  def apply[M[_]: Sync: Parallel: _cost](
```

Comment:
Cool, so this allows overloading, right?

Reply:
Yes, this works!